en/03_Drawing_a_triangle/02_Graphics_pipeline_basics/01_Shader_modules.adoc

```diff
@@ -219,11 +219,12 @@ Create a `compile.sh` file with the following contents:
 
 [,bash]
 ----
-/home/user/VulkanSDK/x.x.x.x/x86_64/bin/glslc shader.vert -o vert.spv
-/home/user/VulkanSDK/x.x.x.x/x86_64/bin/glslc shader.frag -o frag.spv
+glslc shader.vert -o vert.spv
+glslc shader.frag -o frag.spv
```

Comment:
assuming it's in the path, which it might not be (one can install via tarball and not setup the path).
Instead this would work:
$VULKAN_SDK/x86_64/bin/glslc
You can confirm it's in the path with which i.e.
which glslc

Reply:
Thanks, will incorporate this :)